activation/activation.go

```diff
@@ -559,3 +560,29 @@ func (b *Builder) GetPositioningAtx(epochId types.EpochId) (*types.ActivationTxH
 	}
 	return posAtx, nil
 }
+
+// ValidateSignedAtx extracts public key from message and verifies public key exists in IdStore, this is how we validate
```

Comment:
change comment

Reply:
sure but this was the original comment you put in types...
(this was moved here from types to resolve circular dependencies)